frontend/src/i18n/config.ts

```diff
@@ -11,9 +11,9 @@ type RequestConfig = Awaited<
  * List of languages supported by the application. Other tools (Storybook, tests) reference this.
  * These must be BCP47 language tags: https://en.wikipedia.org/wiki/IETF_language_tag#List_of_common_primary_language_subtags
  */
-export const locales = ["en", "es"] as const;
+export const locales = ["en", "es"];
```

Comment:
I'm of two minds here. It's important to list our supported locales, but as far typing is concerned, I am not sure we need to assert that the locale that comes through will always be one of these values. We're doing a check to see if the locale that comes through is supported (see request.ts), and that check seems strange and is complicated TS wise if we are hardening the list of locales as a type here. Fwiw I am fine to revert if we want to go that way